### ISSUE TITLE
Getting React and Redux production build warnings when using CDN-hosted JS bundle

### ISSUE THREAD

**Issue by aaronsarnat** [allenai/hierplane#60]
Currently getting these warnings on Hierplane website that points to new bundle at `<script src="//unpkg.com/hierplane/dist/static/hierplane.min.js"></script>`

![image](https://user-images.githubusercontent.com/8367927/33104902-76e4801e-cedf-11e7-8489-e8dc325dafff.png)

```
hierplane.min.js:1 You are currently using minified code outside of NODE_ENV === 'production'. This means that you are running a slower development build of Redux. You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify or DefinePlugin for webpack (http://stackoverflow.com/questions/30030031) to ensure you have the correct code for your production build.
n.default @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
o.243../applyMiddleware @ hierplane.min.js:1
r @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
o.81../wrapMapToProps @ hierplane.min.js:1
r @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
o.80.../components/connectAdvanced @ hierplane.min.js:1
r @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
o.87../components/Provider @ hierplane.min.js:1
r @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
o.18.../stores/modules/ui @ hierplane.min.js:1
r @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
o.4../EmptyTree.js @ hierplane.min.js:1
r @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
o.10../IconSprite.js @ hierplane.min.js:1
r @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
o.13../Tree.js @ hierplane.min.js:1
r @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
o.33.../module/index.js @ hierplane.min.js:1
r @ hierplane.min.js:1
e @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
(anonymous) @ hierplane.min.js:1
hierplane.min.js:1 Warning: It looks like you're using a minified copy of the development build of React. When deploying React apps to production, make sure to use the production build which skips development warnings and is faster. See https://fb.me/react-minification for more details.
```

**Comment by aaronsarnat** [comment]
@codeviking the code works, just getting warnings. Will let you triage this.

**Comment by codeviking** [comment]
Good catch!

**Comment by codeviking** [comment]
Should be fixed as of v0.0.9.  LMK if that's not the case!
